meetings/working-groups/field-keyword/FK-2024-08-07.md

```diff
@@ -0,0 +1,125 @@
+# `field` keyword working group notes for August 7, 2024
+
+The working group met to review the [Nullability analysis with the `field` keyword](FK-2024-08-07%20Nullability%20analysis%20with%20the%20`field`%20keyword.md) proposal to discuss the direction and come away ready to start on spec updates and present to LDM.
+
+## Takeaways
+
+There was agreement on the changes laid out in the document. This included taking the proposed version of null-resilient getters, instead of the alternatives:
+
+- We like the version of null-resilient getters that take all nullability warnings into consideration, not just ensuring that exit points return non-null expressions. This should be very cheap to implement.
+
+- We see no reason to stop halfway and apply only `[field: MaybeNull]` without `[field: AllowNull]`. The latter would just be harder to explain to users.
+
+### Motivation takeaway
+
+For the motivation to apply nullability automatically, we felt the motivation was much stronger than simply that it is too much code to write to manually specify `[field: MaybeNull]` in a common scenario. Rather, we feel that these nullability attributes are hard for users to understand. It's much greater of a conceptual hurdle than we want for how commonplace it will be to write simple lazy or defaulting scenarios.
```

Comment:
```suggestion
For the motivation to apply nullability automatically, we felt the motivation was much stronger than simply that it is too much code to write to manually specify `[field: MaybeNull]` in a common scenario. Rather, we feel that these nullability attributes are hard for users to understand and get right. It's much greater of a conceptual hurdle than we want for how commonplace it will be to write simple lazy or defaulting scenarios.
```